### ISSUE TITLE
[Suggestion] Integration with Savage & Ravage

### ISSUE THREAD

**Issue by SlimF** [TelepathicGrunt/RepurposedStructures#40]
Currently, if you have both Repurposed Structures and Savage & Ravage, the outposts spawned from Repurposed Structures will not have any mobs in them. Also, the outposts themselves will follow the vanilla design instead of the one from Savage & Ravage. My suggestion is when both mods are installed, Repurposed Structures would spawn outposts with the Savage & Ravage design.

I suppose illagers not spawning in outposts made by RS when R&S is installed counts as a bug, but I found it after I thought of the integration idea, so I'll keep this as a suggestion rather than a bug.

Link for S&R: https://www.curseforge.com/minecraft/mc-mods/savage-and-ravage

**Comment by TelepathicGrunt** [comment]
Looking at this page, it seems they are only on 1.16.1 or older. This does sounds like a bug but I am currently not supporting Minecraft version older than latest. I don’t have the time to handle multiple Minecraft versions. 

Though when they update their mod to 1.16.3+ and if this incompatibility issue still persists, please let me know and I can make a fix for 1.16.3 RS!

**Comment by TelepathicGrunt** [comment]
Oh sorry, I just recalled. The outposts were added in 1.16.3 RS. There’s is no outpost in 1.16.1 RS which is why you cannot find any biome variants of it. Though if illagers are not spawning in the vanilla outposts, you should let them know about that so they can fix it

**Comment by SlimF** [comment]
I should have mentioned I am running a 1.16.4 version of S&R I personally compiled from their Github page. I will let you know if the issue persists once they release the 1.16.4 version officially

**Comment by SlimF** [comment]
@TelepathicGrunt Hello, the official version of S&R has been released and as promised, I am (bugging) reminding you about it. Currently, when you create a new world both S&R and RS's outposts will spawn. S&R changes how the vanilla outposts look while RS vanilla looking outposts, but built with different materials. If it's possible for RS to take S&R's outpost design and add the different building materials to it that would be pretty sweet.

Also, the issue I have mentioned before still persists. RS outposts spawn, but there are no pillagers in them. S&R outposts spawn pillagers around them. I'm not sure if this is a bug on RS or S&R's side, though.

**Comment by TelepathicGrunt** [comment]
I was just talking with someone in my discord and after a quick check, it turns out I forgot to add back the pillager spawning after I did a massive rewrite a long time ago lol. My bad. 

I made an issue report here for myself so I won't forget to fix this when I do a big bugfix for RS after I finish porting Ultra Amplified Dimension. Sorry about that!
https://github.com/TelepathicGrunt/RepurposedStructures-Fabric/issues/68
